src/global/strategy-card-functions.test.js

```diff
@@ -0,0 +1,61 @@
+const assert = require("assert");
+const { globalEvents, MockGameObject } = require("../mock/mock-api");
+const { setupStrategyCard } = require("./strategy-card-functions");
+const TriggerableMulticastDelegate = require("../lib/triggerable-multicast-delegate");
+const locale = require("../lib/locale");
+const { doesNotMatch } = require("assert");
+
+// mock global.js event registration
+globalEvents.TI4 = {
+    onStrategyCardPlayed: new TriggerableMulticastDelegate(),
```

Comment:
We should add the `globalEvents.TI4` setup to mock/mock-api.js, similar to how globals.js adds them.